lib/util/ckan_upload_api.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import json
+
+import requests
+
+log = logging.getLogger(__name__)
+
+
+def upload_to_storage(url, token, file_content, on_progress):
```

Comment:
Again - i don't think we need a separate module for this - either private (or public) methods on client or a single util / helper module would suffice IMO.